tools/checkTheme.js

```diff
@@ -5,7 +5,7 @@ const css = require("css");
 const wcagContrast = require("wcag-contrast");
 const Table = require('cli-table');
 const csscolors = require('css-color-names');
-require("@colors/colors");
```

Comment:
`require("@colors/colors")` <= extends the `String.prototype`, that is the bad practice.